CHANGELOG.rst

```diff
@@ -27,17 +27,17 @@ Added
   (#1059)
 - Added an ASCII art circuit visualizer (#909)
 - Added a new kwarg `filename` to
-  `qiskit.tools.visualization.plot_bloch_vector()` to optionally write the
+  `qiskit.terra.tools.visualization.plot_bloch_vector()` to optionally write the
```

Comment:
do we want to change in the the document

Reply:
same below.  i would not change the changelog

Reply:
I will add `Moving qiskit contents to qiskit.terra` as a change. That's all.